Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -77,6 +81,17 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
         navigationItem.rightBarButtonItem = UIBarButtonItem(customView: createNewChannelButton)
         
         collectionView.backgroundColor = uiConfig.colorPalette.background
+
+        if let flowLayout = collectionViewLayout as? UICollectionViewFlowLayout {
+            // The default separator height is the hair height (1 physical px)
+            flowLayout.minimumLineSpacing = 1 / traitCollection.displayScale
+
+            flowLayout.itemSize = UICollectionViewFlowLayout.automaticSize
+            flowLayout.estimatedItemSize = .init(
+                width: collectionView.bounds.width,
+                height: 64
+            )
+        }
```

Comment:
The same here, whenever we need the separator behaviour we will need to cast it and set always the flowLayout properties, repeating this code in multiple places, and also easier to make a mistake and not put a consistent value with the rest of the UI